app/views/accompaniment_leader/activities/_accompaniment_modal.html.erb

```diff
@@ -41,18 +41,20 @@
               <%= render 'shared/scoped_activity_users_details', activity: activity, scope: 'volunteer' %>
             <% end %>
             <% if activity.accompaniment_reports.present? %>
-              <strong>Accompaniment Leader Notes:</strong>
+              <strong>Report Back:</strong>
```

Comment:
@CZagrobelny I'm not sure if we wanted to change this here ... let me know if I should change it back.

Reply:
@latazzajones yes! This is perfect!